_data/San97/San97.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Sneh Desai# No longer than 28 characters
+institution: Sardar Vallabhbhai Patel Institute of Technology, Vasad🚩 # no longer than 58 characters
+profile_pic: sneh1.jpg # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.
+quote: Success comes to those who not only understands but also acts on creating a better tomorrow. # no longer than 100 characters, avoid using quotes(") to guarantee the format remains the same.
```

Comment:
Fix grammatical error? Please do the following:
* Change `understands` to `understand`
* Change `acts` to `act`